src/simulations/tests/set_generator_test.py

```diff
@@ -414,6 +432,62 @@ def test_sequentially_correlated_all_previous_generator_raise_error(self):
           order='random', correlated_sets='not_implemented', universe_size=30,
           shared_prop=0.2, set_sizes=[10] * 3, random_state=rs)
 
+  @parameterized.parameters(
+      (None, [[0, 0, 0, 0, 0, 0], [0, 0, 1, 1]]),
+      (3, [[0, 0, 0], [0, 0, 1, 1]]))
+  def test_homogeneous_multiset_generator_freq_cap(
+      self, freq_cap, expected_multiset_ids_list):
+    set_sizes = [1, 2]
+    freq_rate_list = [5, 1]
+    rs = FakeRandomState()
+    gen = set_generator.HomogeneousMultiSetGenerator(
+        universe_size=4,
+        set_sizes=set_sizes,
+        freq_rate_list=freq_rate_list,
+        freq_cap=3,
+        random_state=rs)
+    expected_multiset_ids_list = [[0, 0, 0], [1, 1, 0, 0]]
```

Comment:
`expected_multiset_ids_list` is parameterized, so we don't need to hardcode it? My understanding is that we need to pass `freq_cap` & `expected_multiset_ids_list` to `HomogeneousMultiSetGenerator` and expect the value to be the same in parameterized params?

Reply:
Thanks for finding this! I've modified the code. PTAL.